helpers/crud/cuentas_contables/comprobante.go

```diff
@@ -1,29 +1,50 @@
 package cuentas_contables
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
 	"strconv"
 
 	"github.com/astaxie/beego"
 	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/movimientos_contables_mid/helpers"
 	e "github.com/udistrital/utils_oas/errorctrl"
 	r "github.com/udistrital/utils_oas/request"
 )
 
 func GetComprobanteById(id string, comprobante interface{}) (outputError map[string]interface{}) {
 	const funcion string = "GetComprobanteById"
+	var fullResponse map[string]interface{}
 	defer e.ErrorControlFunction(funcion+" - Unhandled Error!", strconv.Itoa(http.StatusInternalServerError))
 
 	url := beego.AppConfig.String("CuentasContablesCrudService") + "/comprobante/" + id
-	if resp, err := r.GetJsonTest(url, &comprobante); err != nil || resp.StatusCode != http.StatusOK {
+	if resp, err := r.GetJsonTest(url, &fullResponse); err != nil || resp.StatusCode != http.StatusOK {
 		status := http.StatusBadGateway
 		if err == nil { // resp.StatusCode != http.StatusOK
 			err = fmt.Errorf("undesired status code - %s", http.StatusText(resp.StatusCode))
 			status = resp.StatusCode
 		}
 		logs.Error(err)
-		outputError = e.Error(funcion+" - r.GetJsonTest(url, &comprobante)", err, strconv.Itoa(status))
+		outputError = e.Error(funcion+" - r.GetJsonTest(url, &fullResponse)", err, strconv.Itoa(status))
 	}
+	helpers.LimpiezaRespuestaRefactorBody(fullResponse, &comprobante)
```

Comment:
En caso que haya habido algun error, es posible que no exista el `Body`, quizás con lo siguiente sea suficiente...?
```suggestion
	} else {
		helpers.LimpiezaRespuestaRefactorBody(fullResponse, &comprobante)
	}
```

Reply:
no es necesario, mapea nullo en caso de no existir